kaldb/pom.xml

```diff
@@ -15,12 +15,12 @@
         <http.client.version>4.5.13</http.client.version>
         <http.core.version>4.4.15</http.core.version>
         <protobuf.version>3.21.7</protobuf.version>
-        <grpc.version>1.47.0</grpc.version>
-        <armeria.version>1.17.0</armeria.version>
-        <micrometer.version>1.9.1</micrometer.version>
-        <kafka.version>3.2.0</kafka.version>
-        <jackson.version>2.13.3</jackson.version>
-        <jackson.databind.version>2.12.7</jackson.databind.version>
+        <grpc.version>1.49.1</grpc.version>
```

Comment:
Keeps grpc/micrometer consistent with what ships in Armeria.

Reply:
Please make this a comment in the code so we can update it when we update armeria next.